application/utils/noise_filter/hashing.py

```
@@ -0,0 +1,96 @@
+    """Normalize `text` and return its SHA-256 hex digest (lowercase, 64 chars).
+
+    This is the canonical dedup key for `knowledge_queue.content_hash`.
+    """
+    normalized = normalize_text(text)
+    return hashlib.sha256(normalized.encode("utf-8")).hexdigest()
+
+
+__all__ = [
+    "normalize_text",
+    "compute_content_hash",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)